Tests iOS/Support/XCTestCase+extensions.swift

```diff
@@ -15,6 +15,10 @@ extension XCTestCase {
         wait(for: expectations, timeout: 10)
     }
 
+    func fulfillment(of expectations: [XCTestExpectation]) async {
```

Comment:
This is needed instead of wait starting with Swift 6 but xcode 14.3 has started warning us.